webapi/admin.go

```diff
@@ -197,12 +198,24 @@ func ticketSearch(c *gin.Context) {
 	})
 }
 
+//checkPasswordHash compare hash value of given AdminPass with AdminPass hash.
+func CheckPasswordHash(hash []byte, password string) bool {
+    err := bcrypt.CompareHashAndPassword(hash, []byte(password))
+    return err == nil
+
+}
 // adminLogin is the handler for "POST /admin". If a valid password is provided,
 // the current session will be authenticated as an admin.
 func adminLogin(c *gin.Context) {
 	password := c.PostForm("password")
+	hashedPass, err := db.GetAdminHash()
+	if err != nil {
+		log.Warnf("Not able to get Admin Hash: %w", err)
+	}
+	
+	ok := CheckPasswordHash(hashedPass, password)
 
-	if password != cfg.AdminPass {
+	if !ok {
```

Comment:
Remove empty line space between, both lines of code are very, very related, should go together.